WcaOnRails/spec/views/registrations/export.csv.erb_spec.rb

```diff
@@ -3,27 +3,27 @@
 
 describe "registrations/export.csv.erb" do
   it "renders valid csv" do
-    competition = FactoryGirl.create :competition
-    FactoryGirl.build(
-      :registration,
-      competition: competition,
-      user: nil,
-      accepted_at: Time.now,
+    competition = FactoryGirl.create :competition, :registration_open
+    user = FactoryGirl.create(
+      :user,
       name: "Bob",
-      countryId: "USA",
-      birthYear: 1990,
-      birthMonth: 1,
-      birthDay: 1,
+      country_iso2: "US",
+      dob: Date.new(1990, 1, 1),
       gender: "m",
       email: "bob@bob.com",
+    )
+    FactoryGirl.create(
+      :registration,
+      competition: competition,
+      accepted_at: Time.now,
+      user: user,
       competition_events: [ competition.competition_events.find_by!(event_id: "333") ],
       guests: 1,
-      guests_old: 'jane', # will go away. https://github.com/thewca/worldcubeassociation.org/issues/403
-    ).save!(validate: false)
+    )
     assign(:competition, competition)
     assign(:registrations, competition.registrations)
 
     render
-    expect(rendered).to eq "Status,Name,Country,WCA ID,Birth Date,Gender,333,333oh,Email,Guests,IP\na,Bob,USA,\"\",1990-01-01,m,1,0,bob@bob.com,1 jane,\"\"\n"
+    expect(rendered).to eq "Status,Name,Country,WCA ID,Birth Date,Gender,333,333oh,Email,Guests,IP\na,Bob,USA,,1990-01-01,m,1,0,bob@bob.com,1,\"\"\n"
```

Comment:
Note that `WCA ID` changed from `""` to `` here. This happened because we got rid of the old `def personId` from `Registration`. I believe this is totally acceptable (like null vs ""), and downstream applications should be able to handle this.